website/docs/r/gcp_source.html.markdown

```diff
@@ -16,6 +16,13 @@ resource "sumologic_gcp_source" "terraform_gcp_source" {
   description   = "My description"
   category      = "gcp"
   collector_id  = "${sumologic_collector.collector.id}"
+  authentication {
+    type = "NoAuthentication"
+  }
+
+  path {
+    type = "NoPathExpression"
+  }
```

Comment:
what happens if these are not set?

Reply:
updated the description

Reply:
is that a bug in the code that it's not respecting defaults? also would the source not work (eg. not decode base64) when this happens?